utils/v2/setup.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { getEnvironmentRequiredVars } from "../utils";
+import { User } from "../User";
+import "@mangata-finance/types";
+import { SubmittableExtrinsic } from "@polkadot/api/types";
+import { getApi, initApi } from "../api";
+
+// API
+export let api: ApiPromise;
+
+// Users
+export let keyring: Keyring;
+export let sudo: User;
+
+export type Extrinsic = SubmittableExtrinsic<"promise">;
+
+export const setupApi = async () => {
```

Comment:
maybe move this out of v2?
I like it and want to use it too!

Reply:
lets do a follow up patch to merge v2 into existing structure if all works ok...

Reply:
Once you finish with the fix of sigle.scenarios.tests I'll create another branch and move only productive code. I'm not happy with the idea of having WIP code in the main branch.